### DISCUSSION TITLE
本地部署提示错误

### DISCUSSION THREAD

**Discussion by qtmulinsen** [illacloud/illa-builder#779] [Q&A] [answered]
用cargo install illa本地部署，到最后的时候提示这样的错误，如何解决？
   Compiling smallvec v1.10.0
   Compiling itoa v1.0.5
error: linker `cc` not found
  |
  = note: No such file or directory (os error 2)

error: could not compile `log` due to previous error
warning: build failed, waiting for other jobs to finish...
error: could not compile `futures-core` due to previous error
error: could not compile `parking_lot_core` due to previous error
error: could not compile `quote` due to previous error
error: could not compile `memchr` due to previous error
error: could not compile `serde_derive` due to previous error
error: could not compile `serde` due to previous error
error: could not compile `syn` due to previous error
error: could not compile `libc` due to previous error
error: could not compile `proc-macro2` due to previous error
error: failed to compile `illa v1.2.6`, intermediate artifacts can be found at `            /tmp/cargo-installdw6kZ6`
[root@localhost appsmith]# cd /home/app
[root@localhost app]# ls

**Comment by karminski** [comment, marked as answer]
@qtmulinsen hi, 从日志上来看是系统缺少编译环境导致的，需要您手动安装编译环境。

- ubuntu/debian 请运行: 
```bash
sudo apt update
sudo apt install build-essential -y
```

- centos 请运行:
```bash
yum update
yum install gcc -y
```

**Reply by qtmulinsen** [reply]
谢谢，问题已经解决

**Reply by Bycorer** [reply]
What to do with win10?

**Comment by Bycorer** [comment]
What to do with win10?
